tests/Feature/GenreTest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Tests\Feature;
+
+// use Illuminate\Foundation\Testing\RefreshDatabase;
+
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Tests\TestCase;
+
+class GenreTest extends TestCase
+{
+    use DatabaseMigrations;
+    /**
+     * A basic test example.
+     */
+    public function test_view_all_genres(): void
```

Comment:
In this test you shoul duse a GenreFactory to create a couple of test genres and then do some more jsonAssertions like hasAll and whereAllType to make sure that the format of the results are always correct

Reply:
See this file for an example of what I mean:

[https://github.com/iO-Academy/23-sept-catmode-bookswap-api/blob/79e4b0caddef118336c5e2ed9e54e102a6f53fa0/tests/Feature/BookTest.php](https://github.com/iO-Academy/23-sept-catmode-bookswap-api/blob/79e4b0caddef118336c5e2ed9e54e102a6f53fa0/tests/Feature/BookTest.php)

Specifically the test\_get\_all\_books method